src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -116,24 +116,12 @@
 
         <!-- Cache -->
 
-        <service id="api_platform.metadata.resource.cache.array" class="Symfony\Component\Cache\Adapter\ArrayAdapter" public="false">
-            <argument>0</argument>
-            <argument>false</argument>
+        <service id="api_platform.metadata.resource.cache" parent="cache.system" public="false">
```

Comment:
In fact this is a nothing to do "directly" with Symfony:

* `system` will store the cache on the server (using APC or a file)
* `app` will store the cache on a cache server (like Redis or Memcache)

`system` is better suited for cache that do not need to be shared between servers (like metadata cache).